client.go

```diff
@@ -0,0 +1,836 @@
+package exposed
+
+import (
+	"bufio"
+	"crypto/tls"
+	"errors"
+	"fmt"
+
+	"context"
+	"io"
+	"net"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/cespare/xxhash"
+	"github.com/thesyncim/exposed/encoding"
+	"github.com/thesyncim/exposed/encoding/proto"
```

Comment:
could not import github.com/thesyncim/exposed/encoding/proto (cannot find package "github.com/thesyncim/exposed/encoding/proto" in any of:
	/usr/local/go/src/github.com/thesyncim/exposed/encoding/proto (from $GOROOT)
	/tmp/tmp.C66j59M3ZD/src/github.com/thesyncim/exposed/encoding/proto (from $GOPATH))